doc(release): prepare release notes for OSS 23.04 (#2824)
centreon/centreon-documentation: i18n/fr/docusaurus-plugin-content-docs/version-23.04/releases/centreon-os.mdx

## Centreon Web

### 23.04.10

Release date: `November 17, 2023`

<details open>
  <summary>Enhancements</summary>

- [API] Added service search in hostGroups / serviceGroups endpoints.
- [Configuration] Added the support of BBDO 3.0.1 for Centreon Broker configuration.

</details>

<details>
  <summary>Bug fixes</summary>

> - [API] Fixed an issue with broker output type when creating an output using CLAPI.

Reviewer: ```suggestion
- [API] Fixed an issue with Broker output types when creating an output using CLAPI.
```

> - [API] Fixed an issue with password clearly displayed in Host & Service API response.

Reviewer: ```suggestion
- [API] Fixed a security issue concerning some API responses.
```

> - [Authentication] Fixed authentication conditions that doesn't work with multiple values (SAML/OpenID Connect).

Reviewer: ```suggestion
- [Authentication] Fixed authentication conditions that did not work with multiple values (SAML/OpenID Connect).
```

> - [Authentication] Enabled GUI access option for imported users authenticated through SSO.

Reviewer: ```suggestion
- [Authentication] The "Reach Centreon Front-end" option is now automatically enabled for imported users authenticated through SSO.
```

> - [Authentication] Fixed match with attribute used to identify users.

Reviewer: ```suggestion
- [Authentication] Fixed an issue with the retrieval of attributes used to identify users.
```

> - [Authentication] Reduced logs messages for better understanding.

Reviewer: ```suggestion
- [Authentication] Removed incorrect log message.
```

> - [Core] Fixed access to pages when Symfony debug mode is enabled.

Reviewer: ```suggestion
- [Core] Fixed access to pages when Symfony debug mode was enabled.
```

- [Event Logs] Fixed pagination.

> - [Install] Fixed SQL permissions for centreon database user to work with MBI product.

Reviewer: ```suggestion
- [Install] Fixed SQL permissions to allow the "centreon" database user to use MBI.
```

> - [Install] Fixed SQL issue during update process.

Reviewer: ```suggestion
- [Install] Fixed an SQL issue occurring during the update process.
```

> - [Monitoring] Fixed display on Firefox when information contains single quote.

Reviewer: ```suggestion
- [Monitoring] Fixed a display issue on Firefox that occurred when the Information column in the Resources Status page contained a single quote.
```

> - [Packaging] Updated centreontrapd mode on debian central.

Reviewer: ```suggestion
- [Packaging] Fixed an issue where centreontrapd was configured for a poller rather than for a central on Debian.
```

> - [Recurrent Downtimes] Fixed downtime when it contains a service group and this service group is linked to a service template.

Reviewer: ```suggestion
- [Recurrent Downtimes] Fixed an issue that occurred when a downtime period contained a service group and this service group was linked to a service template.
```

> - [Ressources Status] Fixed list of resources when using h.alias filter.

Reviewer: ```suggestion
- [Ressources Status] Fixed an issue that occurred when using an h.alias filter on the list of resources.
```

> - [Resources Status] Fixed monitoring server filter when user belong to ACL.

Reviewer: ```suggestion
- [Resources Status] Fixed an issue where the "monitoring server" filter did not display all pollers authorized by the user's ACLs.
```

> - [UX] Fixed the closure of top menus in the banner after clicking on a link.

Reviewer: ```suggestion
- [UX] Fixed an issue where top menus in the banner were not closed after clicking on a link.
```

</details>

<details open>
  <summary>Security fixes</summary>

> - [Security] Fixed SQLi in getHostServicesName method.

Reviewer: ```suggestion
- [Security] Fixed SQLi in a method.
```

> - Fixed potential XSS vulnerability in listing of discovered hosts.

Reviewer: ```suggestion
- Fixed potential XSS vulnerability in the list of discovered hosts.
```

## Centreon Collect

### 23.04.5

Release date: `November 17, 2023`

#### Centreon Broker

<details>
  <summary>Bug fixes</summary>

- Fixed a stability issue that could make Broker crash when BAM was enabled and the database was too slow.

> - Events propagation in BAM has been improved and should be faster.

Reviewer: ```suggestion
- Event propagation in BAM has been improved and should be faster.
```
